src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Rules/CannotChangeVisibility.cs

```diff
@@ -24,6 +25,40 @@ public CannotChangeVisibility(RuleSettings settings, IRuleRegistrationContext co
             context.RegisterOnTypeSymbolAction(RunOnTypeSymbol);
         }
 
+        private static Accessibility NormalizeInternals(Accessibility a) => a switch
+        {
+            Accessibility.ProtectedOrInternal => Accessibility.Protected,
+            Accessibility.ProtectedAndInternal or Accessibility.Internal => Accessibility.Private,
+            _ => a,
+        };
+
+        private int CompareAccessibility(Accessibility a, Accessibility b)
+        {
+            if (!_settings.IncludeInternalSymbols)
+            {
+                a = CannotChangeVisibility.NormalizeInternals(a);
+                b = CannotChangeVisibility.NormalizeInternals(b);
```

Comment:
nit: we usually don't specify the class's name from within that class.

```suggestion
                a = NormalizeInternals(a);
                b = NormalizeInternals(b);
```

Reply:
oh that's weird. i think the automated refactoring just did that. my bad